modules/smart-agent_system-common/conf/04-filesystem-inodes.yaml

```diff
@@ -6,10 +6,10 @@ value_unit: "%"
 signals:
   used:
     metric: system.filesystem.inodes.usage
-    filter: filter('state', 'used')
+    filter: "(filter('state', 'used') and not filter('type', 'squashfs'))"
```

Comment:
This sort of filtering is supposed to happen at the collection level (smartagent or otel-collector).

And `squashfs` is already excluded by default.

How are you collecting the metrics?
Do you have a custom configuration?